BeVolt/Periphs/inc/CAN.h

```diff
@@ -23,6 +23,8 @@
 #define CAN_ID_SOC_DATA								(CAN_ID_BPS | 0x06)
 #define CAN_ID_WDOG_TRIGGERED					(CAN_ID_BPS | 0x07)
 #define CAN_ID_ERROR									(CAN_ID_BPS | 0x08)
+#define CAN_ID_MOTORON								(CAN_ID_BPS | 0x09)
+#define CAN_ID_MOTOROFF								(CAN_ID_BPS | 0x0A)
```

Comment:
@ClarkPoon, @dimembermatt What's the difference between these IDs? The code in this commit is only using one.

Reply:
Should only be one CAN ID (0x10A) called Motor Disable (per the CAN BUS IDs spreadsheet on the drive)